packages/fuchsia_remote_debug_protocol/pubspec.yaml

```diff
@@ -0,0 +1,66 @@
+name: fuchsia_remote_debug_protocol
+description: Provides an API to test/debug Flutter applications on remote Fuchsia devices and emulators.
+homepage: http://flutter.io
+author: Flutter Authors <flutter-dev@googlegroups.com>
+
+dependencies:
+  json_rpc_2: 2.0.7
+  process: 2.0.8
+  meta: 1.1.2
+  web_socket_channel: 1.0.7
+  flutter_test:
+    sdk: flutter
+
+dev_dependencies:
+  mockito: 2.2.3
+  test: 0.12.32+1
+
+  args: 1.4.1 # TRANSITIVE DEPENDENCY
```

Comment:
please re-run the script, it was modified recently so that some of these will now end up in the "dependencies" section rather than "dev_dependencies"

Reply:
Done.